src/components/ecl-breadcrumb/src/styles/ecl-breadcrumb-item.scss

```diff
@@ -0,0 +1,3 @@
+.ecl-breadcrumb__ellipsis > .ecl-button__container {
```

Comment:
not sure what this is supposed to do, but for sure we lose some styles about this button, the padding is not the same, the border is not the same, so maybe if we don't understand the reason why they are not working (normally that would happen because of the scoping of the main css, but here we are adding the class that should match the one in the scoped css) i would set them here.